source/Jobbr.ArtefactStorage.FileSystem.Tests/DirectServiceAccessComponent.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Jobbr.ComponentModel.ArtefactStorage;
+using Jobbr.ComponentModel.Registration;
+
+namespace Jobbr.ArtefactStorage.FileSystem.Tests
+{
+    public partial class ProviderRegistrationTests
```

Comment:
Have the class exist on its own.

General note: partial classes are IMHO usually an anti-pattern, as they indicate that the class is too big and needs to be split-up